packit_service/worker/github_handlers.py

```diff
@@ -330,8 +331,28 @@ def handle_pull_request(self):
                 "\nPlease note that the RPMs should be used only in a testing environment."
             )
             self.project.pr_comment(self.event.pr_id, msg)
+
+            # Testing farm is triggered just once copr build is finished as it uses copr builds
```

Comment:
nasty, we finally need to move to fedmsg-based triggering for copr builds

Reply:
Agree, this breaks the design of the packit-service a lot.

It is a temporary solution just for the testing farm prototype.